config.yml

```diff
@@ -2856,10 +3010,55 @@ nodes:
     fields:
       - name: value
         type: node
+        comment: |
+          Represents the left-hand side of the operator, and can be any kind of Ruby expression.
+
+              foo => bar
+              ^^^
       - name: pattern
         type: node
+        comment: |
+          Represents the right-hand side of the operator. The type of the node depends on the expression.
+
+          Anything that looks like a local variable name (including `_`) will result in a `LocalVariableTargetNode`.
+
+              foo => a # This is equivalent to writing `a = foo`
+                     ^
+
+          Using an explicit `Array` or combining expressions with `,` will result in a `ArrayPatternNode`. This can be preceded by a constant.
+
+              foo => [a]
+                     ^^^
+
+              foo => a, b
+                     ^^^^
+
+              foo => Bar[a, b]
+                     ^^^^^^^^^
+
+          If the array pattern contains at least two wildcard matches, a `FindPatternNode` is created instead.
+
+              foo => *, *a
+                     ^^^^^
```

Comment:
This example probably needs to be updated, wait for results in #2915

Reply:
This has been fixed.